app/styles/ooyala/index.scss

```diff
@@ -0,0 +1,2 @@
+@import 'ooyala-v4';
+@import 'article-featured-video';
```

Comment:
created a ticket XW-3321. I'll do it after merge with on scroll behaviour branch